### DISCUSSION TITLE
Tailscale is not working on my IOS devices (IPAD,etc) on Wifi

### DISCUSSION THREAD

**Discussion by stamandr** [hassio-addons/app-tailscale#112] [Q&A]
Tailscale is not working on my IOS devices (IPAD,etc) on Wifi. If I switch to data it works and I can access devices on my LAN. Go back to WiFi and I cannot connect to devices on my LAN. On Android I do not have this problem. Any suggestions on how to get this to work on WiFi only on IOS devices ?

**Comment by frenck** [comment]
Nope, there is not much the add-on can do about it either. (As in, out of scope).
